File: packages/importer/tests/fixtures/walk/source.cs
public class Telemetry {
    public int Uptime { get; set; }
    public string Status;
    private int hidden;
}

public record TelemetryRecord(int Uptime, string Status);

public enum Status { Idle, Active }

public class Data {
    public int[] Numbers { get; set; }
    public List<string> Labels;
}

File: packages/importer/tests/fixtures/walk/source.py
class Telemetry:
    uptime_ms: int
    status: str

class TelemetryAssign:
    uptime_ms: int = 0
    status: str = "ok"

class Data:
    tags: list[int]
    note: Optional[str]
    other: int | None

class Status(Enum):
    Idle = 1
    Active = 2

class Device:
    """Represents a connected device."""
    id: int
    """Unique ID."""